pkgs/by-name/me/metals/package.nix

```
@@ -7,15 +7,15 @@
   setJavaClassPath,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: rec {
   pname = "metals";
   version = "1.5.2";
 
   deps = stdenv.mkDerivation {
-    name = "${pname}-deps-${version}";
+    name = "${finalAttrs.pname}-deps-${finalAttrs.version}";
```

```suggestion
    name = "metals-deps-${finalAttrs.version}";
```
see the tracking issue for pname overuse